org.eclipse.lsp4e.jdt/src/org/eclipse/lsp4e/jdt/LSJavaProposal.java

```diff
@@ -11,18 +11,36 @@
  *******************************************************************************/
 package org.eclipse.lsp4e.jdt;
 
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+
+import org.eclipse.core.runtime.FileLocator;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.Platform;
 import org.eclipse.jdt.annotation.Nullable;
 import org.eclipse.jdt.internal.codeassist.RelevanceConstants;
+import org.eclipse.jdt.internal.ui.JavaPlugin;
```

Comment:
@mickaelistria , should we be avoiding importing internal classes from the JDT or is this something that is not an issue if you are part of the SimRel?

Reply:
Yes, better avoiding it (when possible)